include/jemalloc/internal/arena_stats.h

```diff
@@ -14,12 +14,20 @@ JEMALLOC_DIAGNOSTIC_DISABLE_SPURIOUS
 typedef struct arena_stats_large_s arena_stats_large_t;
 struct arena_stats_large_s {
 	/*
-	 * Total number of allocation/deallocation requests served directly by
-	 * the arena.
+	 * Total number of large allocation/deallocation requests served directly
+	 * by the arena.
 	 */
 	locked_u64_t	nmalloc;
 	locked_u64_t	ndalloc;
 
+#ifdef LIMIT_USIZE_GAP
```

Comment:
This is a single extra u64 counter. We can accept the cost especially since it's going to be used by default later. No need to cover it with `ifdef`